index.html

```diff
@@ -67,59 +67,68 @@ <h2 class="top-subheader">test case management system</h2>
                     bug tracker integration, fast search, powerful access control and external API.
                 </p>
             </div>
-
-            <div class="rTable secondary">
-                <div class="rTableBody">
-                    <div class="rTableRow">
-                        <div class="rTableHead">Everyday testing</div>
-                        <div class="rTableHead">Test management</div>
-                        <div class="rTableHead">Reporting</div>
-                        <div class="rTableHead">Integration</div>
+            <div class="secondary">
+                <div class="col-12">
```

Comment:
What happens if you remove the wrapping col-12 divs? Shouldn't 4 8 col-3 divs have the same visual layout even without the wrapping col-12 div ?